### PR TITLE
Each jobs command is unique, so if the command causes a crash, we'll identify it on a unique random number

### PR DESCRIPTION
**Pull request by flodnv**
@tylerkaraszewski please review

$ https://github.com/Expensify/Expensify/issues/78808

Not tested

### CODE REVIEW

File: plugins/Jobs.cpp [hunk 1]
```diff
@@ -44,6 +44,10 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
     STable& content = command.jsonContent;
     const string& requestVerb = request.getVerb();
 
+    // Each command is unique, so if the command causes a crash, we'll identify it on a unique random number.
+    command.request["crashID"] = to_string(SRandom::rand64());
```
Comment (coleaeason) [inline plugins/Jobs.cpp]: I don’t think this is going to work... this is it going to prevent the same bad command from crashing multiple nodes.

Comment (coleaeason) [reply plugins/Jobs.cpp]: Okay so i rethought about this, and i was wrong.

**Comment by tylerkaraszewski** [comment]
Seems to break the tests.

**Comment by flodnv** [comment]
Updated @tylerkaraszewski, the issue was that I broke the expected tests behavior in the jobs plugin with this PR -- so I updated the test to be more realistic. Let me know what you think 🙇

**Pull request merged**